common/flavorization/wc_title_holders.txt

```diff
@@ -162,149 +132,2232 @@ warlord_female = {
+	top_liege = no
+	governments = { feudal_government clan_government tribal_government }
+	culture_groups = { centaur_group }
+}
+khanat_centaur_kingdom = {
+	type = title
+	tier = kingdom
+	priority = 200
+	governments = { feudal_government clan_government tribal_government }
+	culture_groups = { centaur_group }
+}
+khagan_male_centaur_emperor = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	governments = { feudal_government clan_government tribal_government }
+	culture_groups = { centaur_group }
+}
+khagan_female_centaur_emperor = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	governments = { feudal_government clan_government tribal_government }
+	culture_groups = { centaur_group }
+}
+khaganat_centaur_empire = {
+	type = title
+	tier = empire
+	priority = 200
+	governments = { feudal_government clan_government tribal_government }
+	culture_groups = { centaur_group }
+}
+
+### Taurens ###
+# Feudal
+chieftain_male_tauren_count = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 116
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+chieftain_female_tauren_count = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 116
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+chiefdom_tauren_county = {
+	type = title
+	tier = county
+	priority = 200
+	culture_groups = { tauren_group }
+}
+high_chieftain_male_tauren_duke = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 126
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+high_chieftain_female_tauren_duke = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 126
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+tribe_tauren_duchy = {
+	type = title
+	tier = duchy
+	priority = 200
+	culture_groups = { tauren_group }
+}
+great_chieftain_male_tauren_king = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 146
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+great_chieftain_female_tauren_king = {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 146
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+confederation_tauren_kingdom = {
+	type = title
+	tier = kingdom
+	priority = 200
+	culture_groups = { tauren_group }
+}
+patriarch_male_tauren_emperor = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 201
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+matriarch_female_tauren_emperor = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 201
+	top_liege = no
+	culture_groups = { tauren_group }
+}
+high_confederation_tauren_empire = {
+	type = title
+	tier = empire
+	priority = 200
+	culture_groups = { tauren_group }
+}
+
```

Comment:
So even tauren theocrats are called chiefs?

P.S. Qiraji, dragons, and more races have the same problem.

Reply:
fixed in 17b8037b8